aten/src/ATen/cpu/vml.h

```diff
@@ -108,12 +108,12 @@ static_assert(
 #define IMPLEMENT_VML_MKL_STUB(op, mklop, type, mkltype)                \
   template <>                                                           \
   inline void v##op(type * out, const type * in, int64_t size) {        \
-    int64_t max_mkl_ind = std::numeric_limits<MKL_INT>::max();          \
+    auto constexpr max_mkl_ind = std::numeric_limits<MKL_INT>::max();   \
     if (size <= static_cast<int64_t>(max_mkl_ind)) {                    \
       vm##mkltype##mklop(                                               \
           size, in, out, VML_HA | VML_FTZDAZ_OFF | VML_ERRMODE_IGNORE); \
     } else {                                                            \
-      MKL_INT ind = 0;                                                  \
+      int64_t ind = 0;                                                  \
```

Comment:
Not sure about this change. Seems like MKL_INT was intentional even it causes a bunch of annoying casts.

Reply:
MKL_INT is 32 in my host.  Clang-tidy complains about index types mismatch.